packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -204,6 +209,7 @@ class _ProductQueryPageState extends State<ProductQueryPage>
           title: _AppBarTitle(
             name: widget.name,
             editableAppBarTitle: widget.editableAppBarTitle,
+            searchResult: widget.searchResult,
```

Comment:
Instead of the `AppBar` here, could you use a `SmoothAppBar`, where there is already a subtitle parameter.
But here you notice, that we can tap on the AppBar.
So the idea is to also change change the  `SmoothAppBar` to allow a click listener on it